### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by korratak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
 # devfordev-workshop
+list friend
```
Comment (ppodee17) [inline README.md]: friend must be written as friends

Comment (BevilKunG) [inline README.md]: add my name please, from benz.

Comment (ppodee17) [inline README.md]: Please ad me "P"

Comment (Apirom) [inline README.md]: add mos to list friend

Comment (korratak) [reply README.md]: what u name?

Comment (ppodee17) [reply README.md]: "P" thank you :D

**Pull request merged**